src/asciidocParser.ts

```diff
@@ -284,9 +286,9 @@ export class AsciidocParser {
       adocCmdArgs.push('-a', 'env-vscode')
 
       adocCmdArgs.push('-q', '-B', '"' + baseDir + '"', '-o', '-', '-')
-      const asciidoctor = spawn(adocCmd, adocCmdArgs, options)
+      spawn(adocCmd, adocCmdArgs, options)
 
-      asciidoctor.stderr.on('data', (data) => {
+      this.processor.stderr.on('data', (data) => {
```

Comment:
@danyill Reading this again, I think the code is incorrect. In this case, the `asciidoctor` variable was referring to the command spawned by the `spawn` function.

`stdout` and `stderr` are not available on the Asciidoctor processor and will most likely throw an exception when using the Asciidoctor CLI instead of Asciidoctor.js.